recipe/meta.yaml

```diff
@@ -8,42 +8,61 @@ package:
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/zope.index-{{ version }}.tar.gz
-  sha256: 055c1949b532b618ee39c6d42f44ee85537a2fe3b1cf333e775a144b0c8b4861
+  sha256: 99f1e041771add767428b12967c54bdb2152b17df9233d1671efabdfe01f6524
 
 build:
-  number: 1
-  script: {{ PYTHON }} -m pip install . -vv
+  # no btrees on s390x
+  skip: True  # [s390x]
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
 
 requirements:
   build:
     - {{ compiler('c') }}
   host:
     - pip
     - python
+    - setuptools
+    - wheel
   run:
     - btrees >=4.4.1
     - persistent
     - python
-    - setuptools
     - six
```

Comment:
`six` is not required. They do list it as a requirement [as of 4.0.1](https://github.com/zopefoundation/zope.index/blob/16fa6211b091da91f0e210bfb99c524c01c8bb5f/CHANGES.rst?plain=1#L126) but I believe it has since been removed.

Reply:
Hold that thought:

```
zope-index 6.1 requires btrees, which is not installed.
```

even though there's no mention of `btrees` in the source.

Reply:
yes, it seems that `btrees` is not used.

that message comes from `pip check` which uses the upstream definition of dependencies